Makefile

```diff
@@ -6,7 +6,9 @@ COMMIT ?= $(if $(shell git status --porcelain --untracked-files=no),$(COMMIT_HAS
 CONTAINER_RUNTIME := $(shell command -v podman 2> /dev/null || echo docker)
 PATH := bin:$(PATH)
```

Comment:
move PATH down after the TOOLSDIR var and set PATH := bin:$(TOOLSDIR)/bin:$(PATH)